src/main/java/org/helpberkeley/memberdata/HBParser.java

```diff
@@ -1,5 +1,5 @@
 //
-// Copyright (c) 2020-2021 helpberkeley.org
+// Copyright (c) 2020-2023 helpberkeley.org
```

Comment:
Should this be 2024?

Reply:
Oops.  I wonder if I have done this on any of the few physical bank checks written this year.